src/sentry/middleware/ratelimit.py

```diff
@@ -30,7 +30,12 @@ def above_rate_limit_check(key, limit=None, window=None):
     if limit is None:
         limit, window = get_default_rate_limit()
 
-    return ratelimiter.is_limited(key, limit=limit, window=window)
+    return {
+        "limited": ratelimiter.is_limited(key, limit=limit, window=window),
+        "current": ratelimiter.current_value(key, window=window),
```

Comment:
This looks like it'll make two requests can we   get a method that returns both as a tuple?